### PR TITLE
Refs RHIROS-385 - fixes host keyerror

### PR DESCRIPTION
**Pull request by kgaikwad**
With this commit, it will show correct values for host id and account.
Message object is different for create/updated and delete event.
It seems that these keys were introduced quite sometime ago but issue never caught before.

### CODE REVIEW

**Comment by kgaikwad** [comment]
Reference doc - https://consoledot.pages.redhat.com/docs/dev/services/inventory.html#_delete_event

File: ros/processor/inventory_events_consumer.py [hunk 1]
```diff
@@ -97,21 +107,25 @@ def host_delete_event(self, msg):
...
-            db.session.commit()
 
     def host_create_update_events(self, msg):
         """ Process created/updated message ( create system record, store new report )"""
         self.prefix = "PROCESSING Create/Update EVENT"
         if 'is_ros' in msg['platform_metadata']:
+            LOG.info(
+                'Process a msg for host(%s) belonging to account %s - %s',
+                msg['host']['id'], msg['host']['account'], self.prefix
+            )
```
Comment (apuntamb) [inline ros/processor/inventory_events_consumer.py:128]: I'm not sure IIUC, but does this log msg want to convey something like this:
```suggestion
            LOG.info(
                'Processing inventory events message for host(%s) belonging to account %s - %s',
                msg['host']['id'], msg['host']['account'], self.prefix
            )
```

**Review by apuntamb — COMMENTED** [review]
Thanks for the quick fix @kgaikwad !! :100:  Just added a minor query. Rest it's good to go.

**Comment by apuntamb** [comment]
> Reference doc - https://consoledot.pages.redhat.com/docs/dev/services/inventory.html#_delete_event

This was really helpful! Thanks for sharing.

File: ros/processor/inventory_events_consumer.py (hunk 1 above)
Comment (kgaikwad) [reply ros/processor/inventory_events_consumer.py:128]: As we are already adding a prefix into message, skipping `inventory events` substring. Except inventory, ROS is not listening to any other service for create/updated event. I hope this fine!

**Review by apuntamb — APPROVED** [review]
lgtm :+1:

**Comment by apuntamb** [comment]
Merged, thanks for the fix @kgaikwad !

**Pull request merged**